Frames.podspec

```diff
@@ -15,12 +15,10 @@ Pod::Spec.new do |s|
 
   s.source_files = 'Source/**/*.swift'
   s.exclude_files = "Classes/Exclude"
-  s.resource_bundles = {
-    'Frames' => ['Source/Resources/**/*']
-  }
+  s.resources = 'Source/Resources/**'
 
   s.dependency 'PhoneNumberKit', '3.3.3'
   s.dependency 'CheckoutEventLoggerKit', '1.2.0'
-  s.dependency 'Checkout', "#{s.version}"
+  s.dependency 'Checkout', '4.0.2'
```

Comment:
Why are we hardcoding this now ? ,  "#{s.version}" would have been better right ?

Reply:
4.0.3 did not require changes to Checkout pod. And looking at the roadmap, I don't see us needing to update Checkout pod for a while, so might be simpler to stop releasing Checkout for the sake of syncing numbers.